python/bin/start.sh

```diff
@@ -62,6 +62,30 @@ fi
 if [ -n "${PY_FILES}" ]; then
   OPT_FILES="${OPT_PY_FILES},${PY_FILES}"
 fi
+#if Hbase catalog is passed, then required hbase dependency are copied to staging location and added to jars
+if [ -n "${CATALOG}" ]; then
```

Comment:
add a variable for skipping the docker build if user wants to reuse already available image

Reply:
Added the functionality for the same, however the step highlighted here is only for jar dependency and not for the image build process. Please refer HBASE_SITE_PATH if statement.